addons/l10n_cl/views/report_invoice.xml

```diff
@@ -229,4 +250,69 @@
         </xpath>
     </template>
 
-</odoo>
+    <template id="tax_totals_widget" inherit_id="account.document_tax_totals">
+        <t t-foreach="tax_totals['subtotals']" t-as="subtotal" position="replace">
+            <t t-if="o.country_code != 'CL'">
+                <t t-foreach="tax_totals['subtotals']" t-as="subtotal">
+                    <tr class="border-black o_subtotal">
+                        <td><strong t-esc="subtotal['name']"/></td>
+
+                        <td class="text-end">
+                            <span t-att-class="oe_subtotal_footer_separator" t-esc="subtotal['formatted_amount']"/>
+                        </td>
+                    </tr>
+
+                    <t t-set="subtotal_to_show" t-value="subtotal['name']"/>
+                    <t t-call="account.tax_groups_totals"/>
+                </t>
+            </t>
+            <t t-else="">
+                <t t-set="subtotal_amounts" t-value="o._l10n_cl_get_amounts()"/>
+                <t t-set="withholdings" t-value="o._l10n_cl_get_withholdings()"/>
+                <t t-if="subtotal_amounts['subtotal_amount_taxable'] != 0.0">
+                    <tr class="border-black is-subtotal"><td><strong>Amount Untaxed</strong></td><td class="text-end oe_subtotal_footer_separator" t-esc="subtotal_amounts['subtotal_amount_taxable']" t-options="{'widget': 'monetary', 'display_currency': subtotal_amounts['main_currency']}"/></tr>
```

Comment:
Change this "Amount Untaxed" to "Net Amount".

Reply:
fixed 254506c926